doc/backend/norms-everkuendung-api-v1.yaml

```diff
@@ -0,0 +1,153 @@
+openapi: 3.0.0
+info:
+  title: RIS Norms API für die E-Verkündung
+  version: 1.0.0
+  description: |
+    API für die Datenübertragung von der E-Verkündung zu NeuRIS.
+
+    Über die Schnittstelle zwischen E-Verkündung und NeuRIS werden die Daten aus dem Bundesgesetzblatt I und II ausgetauscht.
+
+    ## Authentifizierung
+
+    Für die Nutzung aller Endpunkte muss ein gültiger Access Token als `Authorization: Bearer <token>` übermittelt werden. Der Token kann über einen [OAuth2 Client Credentials Grant](https://datatracker.ietf.org/doc/html/rfc6749#section-4.4) abgerufen werden. Die notwendigen Informationen (URL, Client Secret) werden vom DigitalService bereitgestellt.
+
+servers:
+  - url: /api/v1
+    description: Backend
+
+paths:
+  /verkuendungen:
+    post:
+      summary: Annahme von Verkündungen.
+      description: |
+        Die Schnittstelle erlaubt das Hochladen eines Normendokumentationspaketes bis zu einer Größe von 200mb. Das Dokumentationspaket liegt in Form eines ZIP-Archivs vor (`bgbl{1|2}_{jahr}_{nr}.zip`). Es enthält folgende Daten:
+
+        - Rechtsetzungsdokument (XML, PDF)
```

Comment:
Not sure if before the eGG is in place, if the e-Verkündung (or better said the structuring provider - Strukturierungsdienstleister) will also send the Rechtsetzungsdokument